vrt/tests.vrt.js

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-disable flowtype/require-valid-file-annotation */
+/* eslint-env node */
+
+const {configureToMatchImageSnapshot} = require('jest-image-snapshot');
+const {mount} = require('../e2e/helpers');
+const config = require('./config.js');
+const {getAllScenarioNames} = require('./utils.js');
+
+const toMatchImageSnapshot = configureToMatchImageSnapshot({
+  customDiffDir: '__vrt_diffs__',
+  diffDirection: 'vertical',
+});
+
+expect.extend({toMatchImageSnapshot});
+
+describe('visual regression tests', () => {
```

Comment:
what's the plan for handling dark theme / LTR? or did I just miss them?

Reply:
i haven't added any of the alternate passes yet (dark, LTR, mobile). i thought i would do that in a subsequent pr (which is why i didn't remove screener).

i figure they will be additional `describe` blocks for iterating over scenarios. i was going to add a couple new url params to the e2e static site app and wrap the scenarios in rtl/dark theme as needed. It'll probably add a few minutes to build.

i could add it to this pr, it'll just take a lil more time- what do you think? then i could also remove screener 🤔

Reply:
should we merge as-is (keeping screener), so we'd have this running on the other branches too, collecting additional feedback? and we could do that in a follow-up pr

Reply:
i think that is a good idea - it won't be too hard to add the other passes but i do worry about unknown issues when this new system is run at a larger scale.

i'll merge this in as-is. i made a few follow up tickets for other passes (dark/ltr/mobile), configuration restructuring, and better reporting, etc.